tg.js

```diff
@@ -1,821 +1,821 @@
+windows.store.wheels.sub Subukan ang iyong kapalaran at paikutin ang Prize wheel para sa isang pagkakataon upang manalo ng ilang mga bagong disenyo!
+windows.store.currency * Ang mga presyo sa USD ay maaaring mai-convert sa iyong lokal na pera sa isang rate ng palitan na tinukoy ng provider ng pagbabayad o bangko
+windows.store.creator Tangkilikin ang inyong paboritong Content Creator o Streamer sa pamamagitan ng 'Creator Code'!
+windows.chall.customs Ang mga hamon o challenge ay hindi nakukumpleto sa mga kustom na laro
+twitch.link Siguradong nais mo bang i-link ang Twitch account?
+twitch.unlink Siguradong nais mo bang i-unlink ang iyong Twitch account?
+windows.clan.leave Siguradong nais mo bang umalis sa hukbo/samahan ninyo?
+windows.clan.delete Siguradong nais mo bang tanggalin ang hukbo/samahan?
+windows.clan.kick Sigurado ka bang nais mong i-kick si {0} at tanggalin sa samahan?
+windows.store.premium.purchase Nais mong bumili ng {0} araw ng Premium?
+windows.store.premium.gift Nais mo bang regaluhin si '{0}' ng {1} na araw ng Premium?
+windows.playerlist.ban Sigurado ka bang nais mong ipagbawal si {0} mula sa larong ito?
+windows.playerlist.flag Sigurado ka bang nais mong I-flag si {0}?
+windows.playerlist.unflag Sigurado ka bang nais mong I-Unflag si {0}?
+windows.playerlist.punish Sigurado ka bang nais mong parusahin ang manlalarong si {0}?
+item.view.open Buksan ang Model Viewer
+item.view.more Iba pang Impormasyon
+item.sell.cant Hindi mabenta ang kagamitan.
+item.sell.quick Sigurado ka bang ibebenta mo ito agad {0}?
+social.profile.stats Kabuoang statistika
+social.profile.class Statistika ng mga Klase
+social.profile.sales Mga Binebenta
+social.profile.sales.market Mga Binebenta sa Merkado
+social.profile.skins Mga Skins o Disenyo [{0}]
+social.profile.skins.none Walang akdang mga Skins o Disenyo mula sa User
+social.profile.games Mga Laro [{0}]
+social.profile.games.none Walang akdang Laro o Mapa mula sa User
+social.profile.assets Mga Assets [{0}]
+social.profile.assets.none Walang inakdang Assets mula sa User
+social.profile.mods Mga Mods [{0}]
+social.profile.mods.none Walang akdang Mods mula sa User
+social.search Resulta ng Paghahanap
+social.market.stats.rare Bihirang Mahanap na Items
+social.market.stats.circulation Nasa Sirkulasyon
+social.market.stats.units Bilang ng item na ito pandaigdig
+social.trades.in Papasok
+social.trades.in.head Pumapasok na Pag-aalok ({0})
+social.trades.out Palabas
+social.trades.out.head Palabas na Pag-aalok ({0})
+social.trades.history Kasaysayan
+social.trades.history.head Kasaysayan ng Pag-kakalakalan ({0})
+social.trades.decline Tanggihan Lahat
+social.trades.message Payagan ang mga Trade Message at Pag-alok sa Settings
+social.trades.warning Wala kang makukuhang kagamitan sa Pag-alok na ito!
+social.trades.accept Nais mong tanggapin ang Pag-alok na ito?
+stats.airdrops KR-Packages
+settings.quality.flapAnim Animasyon ng Sniperflap

```

Comment:
Can you add space between Sniper And flap?

Reply:
you can request a change

Reply:
Done, committed.